src/fights.ts

```diff
@@ -961,7 +962,12 @@ const freeFightSources = [
       adventureMacro($location`The Hidden Bowling Alley`, pygmyMacro);
     },
     true,
-    {}
+    {
+      cost: () =>
+        sumNumbers(
+          $items`divine champagne popper, Louder Than Bomb, tennis ball`.map((i) => garboValue(i))
+        ) / 11,
```

Comment:
```suggestion
        sum(
          $items`divine champagne popper, Louder Than Bomb, tennis ball`, mallPrice
        ) / 11,
```
We want to use mallPrice instead of garboValue because these are items we buy, not sell; ideally what we'd do is we'd also filter them out based on the skills we use in their stead. Also, go ahead and add the price of the bowl of scorptions to it.